### ISSUE TITLE
Error message when uploading; entries with no files created

### ISSUE THREAD

**Issue by emilyhopkins** [ELVIS-Project/elvis-database#137]
This happened last week, too, and I ended up creating like six entries for a piece (deleted them already)

When I click submit, it shows an error message:

![screen shot 2015-10-13 at 4 25 33 pm](https://cloud.githubusercontent.com/assets/13374693/10467360/3adfe71a-71c7-11e5-936a-4ecab6e1cd65.png)

But then if I go search for the piece, it shows up in the database...

![screen shot 2015-10-13 at 4 25 50 pm](https://cloud.githubusercontent.com/assets/13374693/10467379/59c551c4-71c7-11e5-8ced-aba881ee3b47.png)

But has no files attached!
![screen shot 2015-10-13 at 4 27 25 pm](https://cloud.githubusercontent.com/assets/13374693/10467381/5c0cc1f6-71c7-11e5-9663-365e865753fe.png)

This issue is also likely to create duplicates or more as people retry their uploads...

I left this entry as-is in case you want to take a look, even though it has no files. The two times I've noticed it I've been uploading a lot of stuff in one sitting so that might be part of the issue.

**Comment by lexpar** [comment]
Oh weird. This started happening last week? File uploading was totally working a couple weeks ago.

**Comment by emilyhopkins** [comment]
Yes, I first noticed it on Oct. 5. That file also had unicode in the movement title (Prélude) though not the piece title -- certainly supports your unicode problems theory!

**Comment by lexpar** [comment]
How strange. I don't seem to have problems locally when I do this. Maybe the OS the server is running on isn't happy with these unicode symbols in file names? I wonder.

**Comment by emilyhopkins** [comment]
For the record, I've had the same results working from the lab and at home (using Firefox in both places). I'm at home today and cleared my cache before starting and everything...

I did find a workaround, and did a couple experiments:

First, error:

![error](https://cloud.githubusercontent.com/assets/13374693/10487878/21851b9e-7264-11e5-9965-2ca5a3e4c381.png)

Then, I deleted the unicode character and successfully uploaded:

![without unicode](https://cloud.githubusercontent.com/assets/13374693/10487894/2cd0eb04-7264-11e5-9c66-e0116307fd0e.png)

Then updated to put the unicode back in:

![final](https://cloud.githubusercontent.com/assets/13374693/10487901/34216956-7264-11e5-96b0-adc88557e2c1.png)

Then I wondered if I could just update the empty entry created yesterday by adding files (http://database.elvisproject.ca/piece/2932/), but got an error again:

![when adding files to unicode entry](https://cloud.githubusercontent.com/assets/13374693/10487993/ab3aa7c8-7264-11e5-9976-d2c7bacd3ca2.png)

So it's okay with me updating to include unicode characters but won't let me add files to the empty entry.

**Comment by lexpar** [comment]
Thanks for all the pictures Emily - but you don't need *this* many!

**Comment by emilyhopkins** [comment]
that's fair, sorry for the overkill :P

**Comment by emilyhopkins** [comment]
are you tinkering right now? I was trying to upload something else and it's very angry but I'm guessing it's you...?

**Comment by lexpar** [comment]
Yes - tinkering!

**Comment by lexpar** [comment]
I bet from the user point of view this looks really silly because unicode issues were supposed to have been solved a while ago.

But the tricky thing that happened is that the OS running the website can accept only a subset of all the unicode chars. That's why we only ran into this error now - *some* but not *all* unicode file names were okay with the computer running the site.

Anyway I tried to slap a couple band-aids on before class, but it effects a couple different bits and I;ll have to dig in a bit more this afternoon to work it out. Should be fixed today though. Thanks for the very extensive and patient error reporting :)
